corgi/tasks/management/commands/loadpulpdata.py

```diff
@@ -0,0 +1,68 @@
+import sys
+
+from django.core.management.base import BaseCommand, CommandParser
+
+from corgi.core.models import ProductComponentRelation, ProductStream
+from corgi.tasks.brew import fetch_modular_builds, slow_fetch_brew_build
+from corgi.tasks.pulp import logger
+
+
+class Command(BaseCommand):
+
+    help = "Fetch builds for cdn repos"
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument(
+            "stream_name",
+            type=str,
+            help="Fetch builds using cdn_repo for variants configured in stream",
+        )
+        parser.add_argument(
+            "-i",
+            "--inline",
+            action="store_true",
+            help="Schedule build for ingestion inline (not in celery)",
+        )
+        parser.add_argument(
+            "-f",
+            "--force",
+            action="store_true",
+            help="Force ingestion even if the Build exists",
+        )
+
+    def handle(self, *args, **options) -> None:
+        if options["stream_name"]:
```

Comment:
To avoid build_ids being undefined when stream_name isn't given:
```suggestion
        build_ids = None
        if options["stream_name"]:
```

Reply:
sorry, it looks like I had some local changes which I failed to commit last night, will do it soon.